src/Dependencies/Microsoft.DiaSymReader/Version.targets

```diff
@@ -1,7 +1,7 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <RoslynSemanticVersion>1.0.5</RoslynSemanticVersion>
-    <NuGetVersion>$(RoslynSemanticVersion)</NuGetVersion>
+    <NuGetVersion>$(RoslynSemanticVersion).1</NuGetVersion>
```

Comment:
We don't need to undo. This component versions separately from Roslyn. Whenever a new version is needed this file is updated. BTW, I'll need to rev the dll at some point to add portable PDB interface.